api/src/main/scala/uk/ac/wellcome/platform/api/responses/ResultListResponse.scala

```diff
@@ -8,5 +8,7 @@ case class ResultListResponse(
   @JsonProperty("@context") context: String,
   @JsonProperty("type") ontologyType: String = "ResultList",
   pageSize: Int = 10,
+  totalPages: Int = 10,
```

Comment:
I think it makes more sense to code these in another PR. For now we can get results out.